### ISSUE TITLE
CMIP6 data problem with MCM-UA-1-0

### ISSUE THREAD

**Issue by ledm** [ESMValGroup/ESMValCore#1100]
Hi,

the MCM-UA-1-0 data seems to include longitude values outside the 0-360 acceptable range.

On jasmin, the file is:
```
/badc/cmip6/data/CMIP6/CMIP/UA/MCM-UA-1-0/historical/r1i1p1f2/Omon/tos/gn/latest/tos_Omon_MCM-UA-1-0_historical_r1i1p1f2_gn_185001-201412.nc
```

which includes longitude at -0.9375.

The solution is likely to be adding an intersection command to the fix file:
```
cube = cube.intersection(longitude=(0., 360.))
```

**Comment by remi-kazeroni** [comment]
I think this problem is the same as in #527.  Many variables are affected: `tos`, `sos`, `so`, `thetao`, `sithick`,... I'll open a PR to fix that. 
I transfered this data issue to ESMValCore.

**Comment by remi-kazeroni** [comment]
This can be closed now that #1098 is merged
